spec/mxnet/ndarray_spec.rb

```diff
@@ -30,9 +30,9 @@ module MXNet
         end
       end
 
-      context 'when the array is 2D' do
+      context "when the array is 2D" do
         specify do
-          x = MXNet::NDArray.zeros([2,3])
+          x = MXNet::NDArray.zeros([2, 3])
```

Comment:
Inserting a space following a comma is not essential in this issues.
Please undo all the changes like this.